scripts/openstack-quickstart-demosetup

```diff
@@ -264,7 +264,8 @@ EODASHDB
 SESSION_COOKIE_SECURE = CSRF_COOKIE_SECURE = USE_SSL
 EOSEC
     # sync dashboard DB "after" the database is created
-    run_as wwwrun "cd /srv/www/openstack-dashboard; umask 0027; python -m 'manage' syncdb --noinput"
+    run_as wwwrun "cd /srv/www/openstack-dashboard; umask 0027; python -m 'manage' makemigrations"
```

Comment:
AFAIK, horizon has no migrations at all and will not have. Doing the migrate here Im guessing that its due to using database as the session engine, which also means no migrations.